packages/react/src/components/Toast/utils.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { tabIndexHandler } from './utils';
+
+let toast: HTMLElement | null;
+let targets: HTMLElement[];
+
+beforeEach(() => {
+  {
+    /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+  }
+  const { container } = render(
+    <div>
+      <div className="toaster"></div>
+      <div className="zero" tabIndex={0}></div>
+      <div className="minus-one" tabIndex={-1}></div>
+    </div>
+  );
```

Comment:
This will disable this rule for the whole file. We should just be able to disable it where needed:

```suggestion
  const { container } = render(
    <div>
      <div className="toaster"></div>
      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
      <div className="zero" tabIndex={0}></div>
      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
      <div className="minus-one" tabIndex={-1}></div>
    </div>
  );
```